extensions/amp-subscription/0.1/amp-subscription-platform.js

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
The non-exported files in AMP should not start with "amp-" prefix. Let's just do `subscription-platform`.